cid/helpers/quicksight/__init__.py

```diff
@@ -256,9 +261,23 @@ def _safe_int(value, default=None):
                     logger.info(f'Unable to describe template for {dashboardId}, {exc}')
             else:
                 logger.info("Minimum template version could not be found for Dashboard {dashboardId}: {_template_arn}, deployed template could not be described")
+        
+        if _template_arn is None:
```

Comment:
``
```suggestion
        elif _template_arn is None and "data" in _definition:
```

Reply:
need also add `else`to throw a debug message that this dashboard is not template and not definition based